package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "setup-structure",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
+    "build": "webpack"
```

Comment:
- [ ] Running the script npm run start or npm run build fails to start/compile the project. Please ensure you have added the `webpack` package not just `webpack-cli` or `webpack-dev-server`.